app/components/Map.js

```diff
@@ -15,7 +15,7 @@ import baseImg from '../assets/redPin.png';
 import targetImg from '../assets/blackPin.png';
 import { PinCallout } from './PinCallout';
 import PinEditButton from './PinEditButton';
-import { myCurrLoc, currLoc } from '../lib/db/db';
+import * as geoAction from '../lib/orientation/utils';
```

Comment:
@tiffeli Hmm, it looks you're using `setState` to instantiate at least `currLoc` later, so maybe the fix is as simple as changing line 64 to `this.state.currLoc`.